Source/SVGIconImageListBase.pas

```diff
@@ -281,7 +281,7 @@ procedure TSVGIconImageListBase.DPIChanged(Sender: TObject; const OldDPI, NewDPI
 {$IF CompilerVersion > 29}
 function TSVGIconImageListBase.GetCount: Integer;
 begin
-  raise ENotImplemented.Create('needed to be overridden.');
+  raise ENotImplemented.Create('must be overridden.');
 end;
```

Comment:
Just make it abstract.

Reply:
can't because in later versions of delphi it's actually there already as virtual, and I need to ensure that our two descendants override it.